lib/ui/widget/input_field_widget.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+
+class InputFieldWidget extends StatelessWidget {
+  const InputFieldWidget({
+    required this.textHint,
+    this.keyboardType = TextInputType.text,
+    this.textInputAction = TextInputAction.done,
+    this.isPassword = false,
+    this.textSuffixButton,
+    this.textSuffixButtonCallback,
+    super.key,
+  });
+
+  final TextInputType keyboardType;
+  final TextInputAction textInputAction;
+  final String textHint;
+  final bool isPassword;
+  final String? textSuffixButton;
+  final VoidCallback? textSuffixButtonCallback;
+
+  @override
+  Widget build(BuildContext context) {
+    return TextField(
+      style: const TextStyle(color: Colors.white),
+      decoration: InputDecoration(
+          border: OutlineInputBorder(borderRadius: BorderRadius.circular(Dimensions.radiusNormal)),
+          hintText: textHint,
+          filled: true,
+          fillColor: Colors.white.withOpacity(0.18),
+          contentPadding: const EdgeInsets.only(
+            top: Dimensions.paddingNormal,
+            bottom: Dimensions.paddingNormal,
+            left: Dimensions.paddingSmall,
+            right: Dimensions.paddingSmall,
+          ),
```

Comment:
Or we can use `EdgeInsets.symmetric` instead `EdgeInsets.only` for setting vertical and horizontal insets.

```suggestion
          contentPadding: const EdgeInsets.symmetric(
            vertical: Dimensions.paddingNormal,
            horizontal: Dimensions.paddingSmall,
          ),
```